src/System.Reflection.DispatchProxy/src/CompatibilitySuppressions.xml

```diff
@@ -0,0 +1,54 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I'm not confident I understand the errors as described here: https://learn.microsoft.com/en-us/dotnet/fundamentals/apicompat/diagnostic-ids#list-of-error-codes

@ViktorHofer can you help me understand the suppressions generated with `/p:ApiCompatGenerateSuppresionFile=true`?

Reply:
Which parts exactly?

Reply:
Take lines 18-24: Why are left and right identical? What are we achieving?:

https://github.com/dotnet/maintenance-packages/pull/34/files#diff-d74f5f67d191203f0b95831d53338c5ca7c287aba27e886db861c59deb8598c3R18-R24

Reply:
Lines 32-38: Why are we comparing a ref with a lib?:

https://github.com/dotnet/maintenance-packages/pull/34/files#diff-d74f5f67d191203f0b95831d53338c5ca7c287aba27e886db861c59deb8598c3R32-R38

Reply:
Lines 46-53: Shouldn't PKV006 only trigger with supported .NET versions? It's asking me to add 5.0 and 1.3. Maybe this is something we could consider adding as an improvement to apicompat: Letting the user enabling some feature that will exclude comparisons to unsupported versions.

https://github.com/dotnet/maintenance-packages/pull/34/files#diff-d74f5f67d191203f0b95831d53338c5ca7c287aba27e886db861c59deb8598c3R46-R53